src-cpp/app/modules/shared/section_model.cpp

```diff
@@ -0,0 +1,128 @@
+#include "section_model.h"
+
+namespace Shared
+{
+namespace Models
+{
```

Comment:
if we can use `c++17` we can do `namespace Shared::Models {`

Reply:
Damn I forgot about this awesome thing :) thanks will use it